Readme.md

```diff
@@ -0,0 +1,69 @@
+# userChrome Loader
+This is a template intended to make it easier for end-users to manually load and manage custom CSS.
+
+## Setup Instructions
+1. In your **chrome** folder, save the **userchrome-loader** folder from this repo.
+   
+2. If you already have custom CSS in your **userChrome.css** file, back it up by copying the contents to a new CSS file indside the **userchrome-loader** folder.
+  
+  - If you don't have any custom CSS in your userChrome.css file, skip this step.
+
+3. Copy the **userChrome.css** file from this repo into your own userChrome.css file. 
+
+  - This should be the only content in your userChrome.css file.
```

Comment:
This should use three spaces, not two. In its current state it breaks the formatting.
```suggestion
   - This should be the only content in your userChrome.css file.
```